ics-frontend/src/app/header/header.component.ts

```diff
@@ -1,13 +1,19 @@
 import { Component } from '@angular/core';
-
-
+import { Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'ics-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent {
+  @Output() otherThemeEvent = new EventEmitter<boolean>();
+  otherTheme = false;
```

Comment:
Instead of having this defined both inside the `app.component` and `header.component` you can get it as an `@Input()` inside `header` from `app`

Reply:
@asalov I really don't get how in this case